pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.black]
+line-length = 79
+exclude = '''
+(
+    /(
+        .git
+     |  .github
+     |  docs
+     |  venv
+     |  worlds
+     )/
+)
+'''
+
+[tool.isort]
```

Comment:
@Adman just out of curiosity, how did you come up with these?

Reply:
@mrshu Corporate environment taught me.